libpimeval/src/pimParamsPerf.cpp

```diff
@@ -576,6 +583,9 @@ pimParamsPerf::getMsRuntimeBitSerial(PimDeviceEnum deviceType, PimCmdEnum cmdTyp
         if (it3 != it2->second.end()) {
           const auto& [numR, numW, numL] = it3->second;
           msRuntime += m_tR * numR + m_tW * numW + m_tL * numL;
+          mjEnergy += m_eR * numR + m_eL *numL;
+          mjEnergy += m_pB * msRuntime ;
+          mjEnergy *= 8 * numRanks;  // todo: 8 device per rank, better to get this number from a function in the future
```

Comment:
This does not seem correct but it's okay if you'd like to check in and fix it incrementally. For small vector that does not use so many subarrays, I don't think we need to time it by 8 * numRanks